src/main/scala/it/pagopa/interop/purposeprocess/api/impl/PurposeVersionActivation.scala

```diff
@@ -162,10 +162,8 @@ final case class PurposeVersionActivation(
   }
 
   def getTenantDescription(tenantId: UUID)(implicit contexts: Seq[(String, String)]): Future[String] = for {
-    tenant      <- tenantManagementService.getTenant(tenantId)
-    selfcareId  <- tenant.selfcareId.toFuture(MissingSelfcareId)
-    institution <- partyManagementService.getInstitutionById(selfcareId)
-  } yield institution.description
+    tenant <- tenantManagementService.getTenant(tenantId)
+  } yield tenant.name
```

Comment:
`tenantManagementService.getTenant(tenantId).map(_.name)`

I would also rename the function to `getTenantName` for clarity